olive/engine/engine.py

```diff
@@ -361,6 +359,9 @@ def run_accelerator(
             logger.warning(f"Failed to run Olive on {accelerator_spec}: {e}", exc_info=True)
             return None
 
+        self.footprints[accelerator_spec].to_file(output_dir / f"{prefix_output_name}_footprints.json")
```

Comment:
>output_dir / f"{prefix_output_name}_footprints.json") [](http://example.com/codeflow?start=50&length=53)

better add logging to write which files for future troubleshooting like [https://github.com/microsoft/Olive/pull/919](https://github.com/microsoft/Olive/pull/919)